draft-ietf-httpbis-cache-digest.md

```diff
@@ -263,8 +276,8 @@ we can determine whether there is a match in the digest using the following algo
 
 # IANA Considerations
 
-This draft currently has no requirements for IANA. If the CACHE_DIGEST frame is standardised, it
-will need to be assigned a frame type.
+This draft currently has no requirements for IANA.
+If the specification is standardised, the CACHE_DIGEST frame will need to be assigned a frame type and the Cache-Digest header will need to be registered.
```

Comment:
I'm afraid this isn't adequate; if an RFC -- even an experimental one -- defines a header, it really needs to register it. I can do a separate PR if you like.